### PR TITLE
Add favourites

### LINKED ISSUE #3: Add favourites
As a user I can favourite a contact so that I can see at a glance which contacts are of greater importance to me at a given point in time.

### PR DESCRIPTION
**Pull request by takufunkai**
Fixes #3

### CODE REVIEW

File: src/main/java/seedu/address/model/person/Favourite.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+     * The first character of the address must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    public static final String VALIDATION_REGEX = "[^\\s].*";
+
+    public final String value;
+
+
+    /**
+     * Constructs a {@code Phone}.
```
Comment (weijuey) [inline src/main/java/seedu/address/model/person/Favourite.java:21]: I think you forgot to change this 😆

**Review by weijuey — CHANGES_REQUESTED** [review]
LGTM, just a small nit to fix 👍

File: src/main/java/seedu/address/model/person/Favourite.java (hunk 1 above)
Comment (takufunkai) [reply src/main/java/seedu/address/model/person/Favourite.java:21]: Thanks! Will make the change.

**Review by weijuey — COMMENTED** [review]
LGTM 👍

**Review by weijuey — APPROVED** [review]

**Pull request merged**